app/src/androidTest/java/org/mozilla/tv/firefox/README.md

```diff
@@ -0,0 +1,12 @@
+# UI & Integration tests README
```

Comment:
Should this maybe go into a docs/arch/ADR?

Reply:
Maybe: I'll file a follow-up.

Reply:
Filed https://github.com/mozilla-mobile/firefox-tv/issues/1341